travis/travis_install_nightly

```diff
@@ -68,7 +68,11 @@ fi
 # Workaround to force using system site packages (see https://github.com/Shippable/support/issues/241#issuecomment-57947925)
 rm -f $VIRTUAL_ENV/lib/python2.7/no-global-site-packages.txt
 pip install --upgrade pip
-pip install --user -q --no-binary pycparser -r $(dirname ${BASH_SOURCE[0]})/requirements.txt
+# Remove lxml from odoo requirements because is installed from apt package
+sed -i '/^lxml\=\=/d' ${ODOO_PATH}/requirements.txt
+# Remove python-ldap from odoo requirements because is not a common module used
+sed -i '/^python-ldap\=\=/d' ${ODOO_PATH}/requirements.txt
```

Comment:
`time pip install python-ldap==2.4.15`

```bash
real	0m5.831s
user	0m2.176s
sys	0m0.456s
```

That spent 5 seconds really is not too much but I'm using the same current behaviour where is currently is commented:
 - https://github.com/OCA/maintainer-quality-tools/blob/530d4c09e23b0de52d5cf01e222c16820b592fc3/travis/requirements.txt#L23